Formats/Pioneer/metas.json

```diff
@@ -1,16 +1,15 @@
 {
    Metas:
-  [
-      { "StartDate": "2019-12-13", "Name": "LastOkoDays" },
+   [
       { "StartDate": "2020-01-17", "Name": "PostTherosBeyondDeath" },
-      { "StartDate": "2020-03-11", "Name": "PostOuatBan" },
       { "StartDate": "2020-04-17", "Name": "PostIkoria" },
-      { "StartDate": "2020-06-04", "Name": "PostCompanionChange" },
-      { "StartDate": "2020-06-25", "Name": "PostM21" }, 
+      { "StartDate": "2020-06-25", "Name": "PostM21" },
       { "StartDate": "2020-09-16", "Name": "PostZendikarRising" },
       { "StartDate": "2021-01-28", "Name": "PostKaldheim" },
       { "StartDate": "2021-02-17", "Name": "PostUroBan" },
```

Comment:
This one is a bit weird because we can use two different dates.

Effective Date: February 15, 2021
Cascade rule effective date for Magic Online: February 17, 2021

I think the correct is to use Feb 15 for Pioneer since the Cascade change doesn't apply there, and for Feb 17 for Modern and Legacy.